File: C#/백준/Silver/1436. 영화감독 숌/영화감독 숌.cs
using System;

namespace q{
    class w{
        static void Main(){
            int a=int.Parse(Console.ReadLine()); int i=1; int sum=0;
            while(true){
                string b = i.ToString();
                if(b.Contains("666")){
                    sum++;
                    if(sum==a){
                        Console.WriteLine(i);
                        break;
                    }
                }
                i++;
            }
        }
    }
}

File: Java/백준/Silver/1436. 영화감독 숌/영화감독 숌.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt(); int i=666; int cnt=0;
        while(true){
            String b = Integer.toString(i);
            if(b.contains("666")){
                cnt++;
                if(cnt==a){
                    System.out.println(b);
                    break;
                }
            }
            i++;
        }
    }
}